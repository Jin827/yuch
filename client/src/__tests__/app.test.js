// import React from 'react';
// import { render } from './setupTests';
// import App from '../../app';
// import Loader from '../shared/loader';

// describe('<App />', () => {
//   const Nav = Loader({
//     loader: () => import('../components/nav' /* webpackChunkName: 'Nav' */),
//   });
//   const NavWrapper = render(<Nav />);

//   it('renders correctly with Nav on index page', () => {
//     const props = {
//       history: { location: { pathname: '/' } },
//       Nav: NavWrapper,
//     };
//     const wrapper = render(<App {...props} />);
//     expect(wrapper).toMatchSnapshot();
//   });

//   describe('on sub pages', () => {
//     const props = {
//       history: { location: { pathname: '/login' } },
//       Nav: NavWrapper,
//     };
//     const wrapper = render(<App {...props} />);

//     it('contains Loadable Component', () => {
//       expect(wrapper.find('LoadableComponent').exists()).toBe(true);
//     });

//     it('renders correctly with Nav', () => {
//       expect(wrapper).toMatchSnapshot();
//     });
//   });
// });
