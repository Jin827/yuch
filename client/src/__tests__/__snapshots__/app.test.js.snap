// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> on sub pages renders correctly with Nav 1`] = `
<div
  id="app"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charset="UTF-8"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <title>
      위탁급식/식당 유청
    </title>
    <meta
      content="위탁급식/식당 유청"
      property="og:title"
    />
    <meta
      content="착한가격의 집밥, 위탁 급식, 뷔폐 식당, 기업체 각종 행사/ 모임 단체 식사, 경주 전지역 배달, No MSG, 신선한 지역 농산물"
      property="og:description"
    />
    <meta
      content="###"
      property="og:type"
    />
    <meta
      content="###"
      property="og:url"
    />
    <meta
      content="./../assets/img/header.jpg"
      property="og:image"
    />
    <link
      href="http://domain/humans.txt"
      rel="author"
      type="text/plain"
    />
    <link
      href="http://domain/robots.txt"
      rel="author"
      type="text/plain"
    />
  </HelmetWrapper>
  <LoadableComponent />
</div>
`;

exports[`<App /> renders correctly with Nav on index page 1`] = `
<div
  id="app"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charset="UTF-8"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <title>
      위탁급식/식당 유청
    </title>
    <meta
      content="위탁급식/식당 유청"
      property="og:title"
    />
    <meta
      content="착한가격의 집밥, 위탁 급식, 뷔폐 식당, 기업체 각종 행사/ 모임 단체 식사, 경주 전지역 배달, No MSG, 신선한 지역 농산물"
      property="og:description"
    />
    <meta
      content="###"
      property="og:type"
    />
    <meta
      content="###"
      property="og:url"
    />
    <meta
      content="./../assets/img/header.jpg"
      property="og:image"
    />
    <link
      href="http://domain/humans.txt"
      rel="author"
      type="text/plain"
    />
    <link
      href="http://domain/robots.txt"
      rel="author"
      type="text/plain"
    />
  </HelmetWrapper>
</div>
`;
